drivers/storage/portworx/testspec/prometheusRule.yaml

```diff
@@ -29,19 +29,6 @@ spec:
       labels:
         issue: Portworx volume {{$labels.volumeid}} usage on {{$labels.instance}} is critical.
         severity: warning
-    - alert: PortworxVolumeWillFill
```

Comment:
just curious why is this removed? is the prediction inaccurate?

Reply:
https://portworx.atlassian.net/browse/PD-904